plugins/katello/3.4/release_notes/release_notes.md

```diff
@@ -4,12 +4,28 @@ title: Release Notes
 version: 3.4
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.4 (Oud bruin) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.4/CHANGELOG.md).
 
 ## Features
 
+#### File Repository Additions
+
+#### Repository Force Sync
```

Comment:
Two new options are now present when syncing to help alleviate two issues that users occasionally encounter.   These options are available under the 'Advanced Sync' option under the repository details page and via Bulk Product selection.  These issues are:

1) I do not see content locally that is in the upstream repo and resyncing shows nothing new to sync (Complete Sync)
2) Some rpm has become corrupt or been deleted locally and i need to force them to be resynced (Validate Content Sync)